plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableSearchQuery.php

```diff
@@ -34,6 +41,25 @@ class OrdersTableSearchQuery {
 	public function __construct( OrdersTableQuery $query ) {
 		$this->query       = $query;
 		$this->search_term = "'" . esc_sql( '%' . urldecode( $query->get( 's' ) ) . '%' ) . "'";
+
+		$this->search_fields = array_map(
+			'wc_clean',
+			/**
+			 * Compatibility for similarly named filter in the CPT data store.
+			 * If someone wants to query only billing email, phone, then fields query is a better option, so we don't list it here.
```

Comment:
Similar to an earlier note, we could (optionally) mark this as a duplicate hook:

```suggestion
			 * This filter is documented in ../../../../includes/data-stores/class-wc-order-data-store-cpt.php
```

Though if it is important to provide additional HPOS-specific context, then feel free to ignore.